src/install.ts

```diff
@@ -4,48 +4,24 @@ import {mv} from '@actions/io'
 import {chmod} from '@actions/io/lib/io-util'
 import * as tc from '@actions/tool-cache'
 import * as exec from '@actions/exec'
-import {execWithOutput} from './exec'
 
-const CERT_IDENTIFIER = 'Developer ID Installer: AgileBits Inc. (2BUA8C4S2C)'
 const KEY_FINGERPRINT = '3FEF9748469ADBE15DA7CA80AC2D62742012EA22'
 
 export async function install(onePasswordVersion: string): Promise<void> {
   const platform = os.platform().toLowerCase()
 
-  let extension = 'zip'
+  let arch = 'amd64'
   if (platform === 'darwin') {
-    extension = 'pkg'
+    arch = 'arm64'
   }
-  const onePasswordUrl = `https://cache.agilebits.com/dist/1P/op/pkg/v${onePasswordVersion}/op_${platform}_amd64_v${onePasswordVersion}.${extension}`
-  const archive = await tc.downloadTool(onePasswordUrl)
+  const onePasswordUrl = `https://cache.agilebits.com/dist/1P/op2/pkg/v${onePasswordVersion}/op_${platform}_${arch}_v${onePasswordVersion}.zip`
   core.info(
     `Downloading ${onePasswordVersion} for ${platform} from ${onePasswordUrl}`
   )
+  const archive = await tc.downloadTool(onePasswordUrl)
+  const extracted = await tc.extractZip(archive)
 
-  let extracted: string
-  if (platform === 'darwin') {
```

Comment:
Question: Does this mean we are no longer verifying the binary on MacOS? I assume the action itself still functions as expected on the darwin platform?

Reply:
I had to switch to using the .zip file because the .pkg no longer has a `/Payload` file given that the pkg version is now a (Mac) universal distribution.

I spent some time trying to figure out if we could still do verification on the zip file but wasn't able to find a solution. If in the future we are able to do this type of verification it would be great to add it back.